lib/admin/db_drift.dart

```diff
@@ -516,7 +517,7 @@ class MyDatabase extends _$MyDatabase {
       }
       throw Exception('Failed to load assets from Maximo');
     }
-    print(result['member'].length);
+    material.debugPrint(result['member'].length);
```

Comment:
the debugPrint causes an error since you are printing an int

any reason for using debugPrint as opposed to just print?

seems a bit extra to import material library into the db file

Reply:
want to remove as many "problems" from the code as I can. Will change it to _result['member'].length.toString()_